### ISSUE TITLE
False positive SlevomatCodingStandard.TypeHints.DeclareStrictTypes.IncorrectWhitespaceBeforeDeclare: Expected 0 lines before declare statement, found 0.

### ISSUE THREAD

**Issue by LastDragon-ru** [slevomat/coding-standard#1206]
It 100% worked in dev-master a278c68 but [somewhere since](https://github.com/LastDragon-ru/lara-asp/actions/workflows/phpcs.yml) dev-master 0858034 it shows an error:

```txt
> @php ./vendor/bin/phpcs 'packages/Bug.php'

FILE: /project/packages/Bug.php
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 1 | ERROR | [x] Expected 0 lines before declare statement, found 0.
----------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------

Time: 185ms; Memory: 10MB
```

```php
<?php declare(strict_types = 1);

namespace Test;

class Bug {
    // empty
}
```

```xml
<?xml version="1.0"?>
<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:noNamespaceSchemaLocation="./vendor/squizlabs/php_codesniffer/phpcs.xsd">
    <config name="installed_paths" value="../../slevomat/coding-standard"/>
    <file>./packages</file>
        <rule ref="SlevomatCodingStandard.TypeHints.DeclareStrictTypes">
        <properties>
            <property name="declareOnFirstLine"/>
            <property name="linesCountBeforeDeclare" value="0"/>
            <property name="spacesCountAroundEqualsSign" value="1"/>
        </properties>
    </rule>
</ruleset>
```

**Comment by kukulich** [comment]
@LastDragon-ru Your configuration is invalid. There’s no value for declareOnFirstLine

**Comment by LastDragon-ru** [comment]
@kukulich but it is in docs :)

![image](https://user-images.githubusercontent.com/1329824/115989669-7df63480-a5d0-11eb-8622-6841bf6c58c4.png)

**Comment by kukulich** [comment]
@LastDragon-ru yes, the option exists but you configured it wrong.

**Comment by LastDragon-ru** [comment]
@kukulich mmmm but it works ... 

![image](https://user-images.githubusercontent.com/1329824/115989753-ef35e780-a5d0-11eb-92f2-ac3f76877014.png)

**Comment by LastDragon-ru** [comment]
Ah, I see, ok, this:

```
<?xml version="1.0"?>
<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:noNamespaceSchemaLocation="./vendor/squizlabs/php_codesniffer/phpcs.xsd">
    <config name="installed_paths" value="../../slevomat/coding-standard"/>
    <file>./packages</file>
    <rule ref="SlevomatCodingStandard.TypeHints.DeclareStrictTypes">
        <properties>
            <property name="linesCountBeforeDeclare" value="0"/>
            <property name="spacesCountAroundEqualsSign" value="1"/>
        </properties>
    </rule>
</ruleset>
```

Still show weird:

```
> @php ./vendor/bin/phpcs 'packages/Bug.php'

FILE: /project/packages/Bug.php
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 1 | ERROR | [x] Expected 0 lines before declare statement, found 0.
----------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------
```

**Comment by LastDragon-ru** [comment]
Finally solved 😁

```
    <rule ref="SlevomatCodingStandard.TypeHints.DeclareStrictTypes">
        <properties>
            <property name="declareOnFirstLine" value="true"/>
            <property name="spacesCountAroundEqualsSign" value="1"/>
        </properties>
    </rule>
```
